README.md

```diff
@@ -58,10 +58,10 @@ export default class FeatureDisplay extends Component {
   render() {
     return (
       <Feature flagId="testFlagId">
-        <Variant variation={ true }>
+        <Variant variation={true}>
```

Comment:
i thought we decided to keep the spaces?

Reply:
Stupid auto format on save